counterpartylib/lib/messages/issuance.py

```diff
@@ -306,6 +318,13 @@ def parse (db, tx, message):
     fee = 0
     if status == 'valid':
         call_date, call_price, problems, fee, description, divisible, reissuance, reissued_asset_longname = validate(db, tx['source'], tx['destination'], asset, quantity, divisible, callable_, call_date, call_price, description, subasset_parent, subasset_longname, block_index=tx['block_index'])
+
+        # extra validation for parsing only.  
+        # A reissuance description that contains the string ;;l in the description is invalid
+        if not problems and reissuance and ';;l' in unmodified_description and util.enabled('subassets', block_index=tx['block_index']):
+            problems.append('reissuance description cannot contain ;;l')
```

Comment:
`;` is used to separate multiple problems in the resultant status string (as a few lines below this one shows), of course. ...I'm concerned that some programs may be parsing out multiple problems by splitting at `;`... maybe instead of `'reissuance description cannot contain ;;l'` do something like `'reissuance description cannot re-specify longname key-value pair'` or something similar